packages/log/src/log.ts

```diff
@@ -255,12 +255,19 @@ export class Log {
     }
 
     /**
-     * Logs given message only once as WARNING. It's used to warn user that some feature he is using has been deprecated.
+     * Logs a `WARNING` level message only once.
      */
-    deprecated(message: string) {
-        if (this.deprecationsReported[message]) return;
+    warningOnce(message: string) {
+        if (this.warningsOnceLogged.has(message)) return;
 
-        this.deprecationsReported[message] = true;
+        this.warningsOnceLogged.add(message);
         this.warning(message);
     }
+
+    /**
+     * Alias for log.warningOnce() kept for backwards compatibility
```

Comment:
lets keep the comment as it was, this method is not just alias for BC, its valid public API 

```suggestion
     * Logs given message only once as WARNING. It's used to warn user that some feature he is using has been deprecated.
```